templates/frontend/pages/userLostPassword.tpl

```diff
@@ -27,6 +27,7 @@
 		{/if}
 
 		<fieldset class="fields">
+			<legend>{translate key="user.email"}</legend>
```

Comment:
Instead of adding a `<legend>`, I think we can change the `<fieldset>` to `<div>`. There's only one input field here so the `<fieldset>` does not describe multiple fields.